core/src/main/java/kafka/server/share/SharePartitionManager.java

```diff
@@ -155,7 +155,7 @@ public SharePartitionManager(
         );
     }
 
-    SharePartitionManager(
+    private SharePartitionManager(
```

Comment:
This should be public as it will be used in BrokerServer.scala.

Reply:
no, the constructor that is utilized in BrokerServer.scala is present `here`, so we can let this one remain private